python/ray/runtime_context.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import ray.worker
+import json
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class RuntimeContext(object):
+    """A class used for getting runtime context."""
+
+    def __init__(self, worker=None):
+        self.worker = worker
+
+    @property
+    def current_job_id(self):
+        """Get current driver ID for this worker or driver.
```

Comment:
Can you either change the method name to `driver_id` or change the docstring to use the term job_id instead of driver id?